src/Fantomas.Core/CodePrinter2.fs

```diff
@@ -3351,7 +3349,12 @@ let genTypeDefn (td: TypeDefn) =
                  +> sepNlnTypeAndMembers typeDefnNode
                  +> genMemberDefnList members)
                     ctx
-            else
+
+            match ctx.Config.MultilineBracketStyle with
+            | Aligned
+            | ExperimentalStroustrup -> aligned
+            | Classic when (List.exists (fun (fieldNode: FieldNode) -> fieldNode.XmlDoc.IsSome) node.Fields) -> aligned
```

Comment:
The original code was already a bit icky but could you extract `List.exists (fun (fieldNode: FieldNode) -> fieldNode.XmlDoc.IsSome) node.Fields` to some value? Call it `anyFieldHasXmlDoc` maybe.